tests/plugins/test_signer.py

```diff
@@ -176,13 +212,15 @@ def test_signer_plugin_rollback(
 def test_signer_plugin_refresh(
     editor_client: RemoteSettingsClient,
     reviewer_client: RemoteSettingsClient,
+    to_review_enabled: bool,
 ):
     resource = signed_resource(editor_client)
     preview_bucket = resource["preview"]["bucket"]
     dest_bucket = resource["destination"]["bucket"]
     upload_records(editor_client, 5)
 
-    editor_client.patch_collection(data={"status": "to-review"})
+    if to_review_enabled:
+        editor_client.patch_collection(data={"status": "to-review"})
```

Comment:
I think you can execute this always, since it's not blocked when review is disabled

Reply:
For whatever reason, that's how the old test worked. [link](https://github.com/mozilla/remote-settings/blob/main/tests/integration/plugins/test_signer.py#L219-L220)

But yeah I can remove it since it's harmless.